Userland/Libraries/LibELF/Arch/aarch64/entry.S

```diff
@@ -8,5 +9,4 @@
 .hidden _invoke_entry
 .type _invoke_entry,@function
 _invoke_entry: # (argc, argv, envp, entry)
-    # FIXME: Possibly incomplete.
-    ret
```

Comment:
Iirc the x86_64 implementation here takes care to ensure alignment of the stack matches the SysV ABI (16 bytes). Do we need a similar guarantee for aarch64?

Reply:
The aarch64 hardware will actually trap if the stack pointer is not 16-byte aligned, and this code works as expected. So that means that hardware actually enforces it, and an exception will be generated once it is not the case. So I think this is fine, and if in the future the codegen of the compiler change, such that it is not aligned anymore, we'll be notified by that because of the hardware enforcement.